admittance_controller/src/admittance_rule.cpp

```diff
@@ -283,7 +295,8 @@ controller_interface::return_type AdmittanceRule::update(
   }
 
   // Add deltas to previously-desired pose to get the next desired pose
-  // TODO: Use convert_to_array method
+  // FIXME: Why not use convert_to_array method?
+  // FIXME: (?) Does this variable have a wrong name? Shouldn't it be target_pose_ik_base_frame?
```

Comment:
I'm fairly certain, the name of `feedforward_pose_ik_base_frame_` is fine. It is the feedforward pose and it's in the `ik_base` frame.

I think the name of `target_ik_tip_deltas_vec` should be changed to `target_ik_base_deltas_vec`. Since it shoul be in `ik_base` frame, too. (L279-280 explains why)

Reply:
renamed.